### ISSUE TITLE
Request to clarify width and height arguments in Image.asset() constructor API docs

### ISSUE THREAD

**Issue by sethladd** [flutter/flutter#10877]
Was reading https://docs.flutter.io/flutter/widgets/Image/Image.asset.html and we mention: "If width and height are both specified, and scale is not, then size-aware asset resolution will be attempted also."

A couple of questions come to mind:

* What if width and height are not specified?
* What do width and height actually mean? The expected w and h of the image? The "scale to this w and h" ?

basically, do w and h mean " the w and h of the image as I expect it actually is " or " the w and h of how I want the image rendered to screen " ?

**Comment by Hixie** [comment]
Turns out it means nothing, that documentation is a lie. We don't yet implement size-aware asset resolution.

**Comment by sethladd** [comment]
:)

**Comment by Hixie** [comment]
https://github.com/flutter/flutter/pull/10904
